### DISCUSSION TITLE
tag helper <style asp-name""> not work in my module

### DISCUSSION THREAD

**Discussion by MRSheikh** [OrchardCMS/OrchardCore#16031] [Q&A]
hi.
I use Orchardcore without cms "OrchardCore.Application.Mvc.Targets" to my projects and add my module to it. 
I know to add 
`builder.Services.AddOrchardCore().ConfigureServices(s =>
{   s.AddResourceManagement();
    s.AddTagHelpers<LinkTagHelper>();
    s.AddTagHelpers<MetaTagHelper>();
    s.AddTagHelpers<ResourcesTagHelper>();
    s.AddTagHelpers<ScriptTagHelper>();
    s.AddTagHelpers<StyleTagHelper>();
});` 
in my program and then implement `ResourceManifest.cs` for my module too. 
after that in `_ViewImports.cshtml` add 
`@addTagHelper *, OrchardCore.DisplayManagement
@addTagHelper *, OrchardCore.ResourceManagement`

and in `_ViewStart.cshtml` use tag helpers like this:

> <style asp-name="bootstrap"></style><style asp-name="site"></style><script asp-name="myjquery"></script><script asp-name="bootstrap"></script><script asp-name="site"></script>

 all scripts were added successfully into my module but nothing for styles????

what should I do to import my module styles to tag helper?

**Comment by MRSheikh** [comment]
> hi. I use Orchardcore without cms "OrchardCore.Application.Mvc.Targets" to my projects and add my module to it. I know to add `builder.Services.AddOrchardCore().ConfigureServices(s => { s.AddResourceManagement(); s.AddTagHelpers<LinkTagHelper>(); s.AddTagHelpers<MetaTagHelper>(); s.AddTagHelpers<ResourcesTagHelper>(); s.AddTagHelpers<ScriptTagHelper>(); s.AddTagHelpers<StyleTagHelper>(); });` in my program and then implement `ResourceManifest.cs` for my module too. after that in `_ViewImports.cshtml` add `@addTagHelper *, OrchardCore.DisplayManagement @addTagHelper *, OrchardCore.ResourceManagement`
> 
> and in `_ViewStart.cshtml` use tag helpers like this:
> 
> > <style asp-name="bootstrap"></style><style asp-name="site"></style><script asp-name="myjquery"></script><script asp-name="bootstrap"></script><script asp-name="site"></script>
> 
> all scripts were added successfully into my module but nothing for styles????
> 
> what should I do to import my module styles to tag helper?

I added `builder.Services.AddOrchardCore();` in my program and reference `OrchardCore.DisplayManagement` in my module too.

**Reply by ns8482e** [reply]
you need to add `<resources type="Header" />` just before `</head>`   and set all your styles to `Header`

**Comment by weirdyang** [comment]
For modules, you need to add a reference to `OrchardCore.ResourceManagement` package
